eZ/Publish/API/Repository/TransactionService.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
```

Comment:
```suggestion
 * @copyright Copyright (C) Ibexa AS (formerly eZ Systems AS). All rights reserved.
```

Please apply this change to other files as well.

Reply:
File header is part of the ezsystems/ezplatform-code-style and needs to be changed there and then applied to whole code base.